composer.json

```diff
@@ -22,13 +22,13 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "beatswitch/lock": "0.1.*"
+        "beatswitch/lock": "0.1.*",
+        "illuminate/auth": "5.0.x",
+        "illuminate/database": "5.0.x",
+        "illuminate/support": "5.0.x"
```

Comment:
Which do you want me to use? .x is the preferred/documented approach on composer docs.